src/garage/torch/algos/maml.py

```diff
@@ -137,6 +143,9 @@ def train_once(self, runner, all_samples, all_params):
                                                   kl_after.item(),
                                                   policy_entropy.mean().item())
 
+        if self._meta_evaluator:
+            self._meta_evaluator.evaluate(self)
```

Comment:
Should it evaluate every n epochs? Evaluate in each epoch seems to introduce a huge overhead, especially when running large benchmarks like ML45.

Reply:
How do you suggest to handle dowel's caveat of not supporting dynamic tabular headers?

Reply:
There will be ONE warning at the beginning. After talking to KR, we think it's fine by now.

Reply:
I suggest a param similar to `evaluate_every_n_epochs` or whichever param name is used in the off-policy algos